Documentation/op-guide/monitoring.md

```diff
@@ -100,7 +100,7 @@ Now Prometheus will scrape etcd metrics every 10 seconds.
 
 ### Alerting
 
-There is a [set of default alerts for etcd v3 clusters](./etcd3_alert.rules).
+There is a set of default alerts for etcd v3 clusters for [Prometheus 1.x](./etcd3_alert.rules) as well as [Prometheus 2.x](./etcd3_alert.rules).
```

Comment:
one of the links is wrong. they point to the same file now. the v2 one should have .yam suffix?

/cc @gyuho @brancz

Reply:
You're absolutely right. I'll fix it with the follow-up PR!

Reply:
Right,

@brancz when you fix alert rules for https://github.com/coreos/etcd/issues/8802, can we also change `[Prometheus 2.x](./etcd3_alert.rules).` to `[Prometheus 2.x](./etcd3_alert.rules.yml).`?

Thanks!